src/sidebar/service-url.js

```diff
@@ -3,44 +3,91 @@
 var urlUtil = require('./util/url-util');
 
 /**
- * A map of all route names to relative URLs on the Hypothesis service that
- * the client links to.
+ * An AngularJS service that returns an absolute URL given a link name and
```

Comment:
Ugh, I moved `service-url.js` to `legacy-service-url.js`, and then put the new `serviceUrl` at `service-url.js`, so git is showing it as a _modified_ file, but it's really completely replaced. You may just want to view the `service-url.js` file, instead of looking at this diff.